src/network/services/force-update.ts

```diff
@@ -0,0 +1,46 @@
+import { Linking, Platform } from 'react-native';
+import DeviceInfo from 'react-native-device-info';
+
+import remoteConfig from '@react-native-firebase/remote-config';
+
+export const isUpdated = (): boolean => {
+  remoteConfig()
+    .setDefaults({
+      minimum_version: '',
+    })
+    .then(() => remoteConfig().fetchAndActivate())
+    .then(fetchedRemotely => {
+      if (fetchedRemotely) {
+        const remoteVersion = remoteConfig().getValue('').asString();
+        const localVersion = DeviceInfo.getVersion();
+        return compareVersions(localVersion, remoteVersion);
+      }
+    });
+  return true;
+};
+
+export const openStore = () => {
+  if (Platform.OS === 'android') {
```

Comment:
Here you can use `Platform.select` to only call `Linking.openURL` 

[Platform select](https://reactnative.dev/docs/platform#select)

Reply:
Done